tests/unittests/onnxImporterTest.cpp

```diff
@@ -430,6 +430,36 @@ TEST(onnx, importSparseToDense) {
   EXPECT_EQ(vals, mod.getPlaceholderByName("values"));
 }
 
+/// Test loading SparseLengthsSum from an ONNX model.
+TEST(onnx, importSparseLengthsSum) {
+  ExecutionEngine EE;
+  auto &mod = EE.getModule();
+  auto *F = mod.createFunction("main");
+  std::string netFilename("tests/models/onnxModels/sparseLengthsSum.onnxtxt");
+  Placeholder *output;
+  {
+    Tensor data(ElemKind::FloatTy, {2, 1});
+    Tensor indices(ElemKind::Int64ITy, {2});
+    Tensor lengths(ElemKind::Int32ITy, {2});
+    ONNXModelLoader onnxLD(
+        netFilename, {"data", "indices", "lengths"},
+        {&data.getType(), &indices.getType(), &lengths.getType()}, *F);
+    output = onnxLD.getSingleOutput();
+  }
+  // Verify structure: PH, PH ->  SparseLengthsSum -> Save -> PH.
+  //                  PH -> Splat /
+  ASSERT_EQ(mod.getPlaceholders().size(), 4);
+  ASSERT_EQ(F->getNodes().size(), 3);
+  auto *save = getSaveNodeFromDest(output);
+  auto *LS =
+      llvm::dyn_cast<SparseLengthsWeightedSumNode>(save->getInput().getNode());
+  ASSERT_TRUE(LS);
+  ASSERT_TRUE(llvm::isa<Placeholder>(LS->getData()));
```

Comment:
nit: those all below could be EXPECT_EQ to cover all the checks for data, indices, weights.